### DISCUSSION TITLE
已提前付款续期2年，但仍然提示即将到期，可用时间没有任何变化，望解决。

### DISCUSSION THREAD

**Discussion by geniuszhangke** [getlantern/lantern-forum-cn#593] [Q&A]
今天下午15：24分付款续期了两年的蓝灯专业版，但仍然提示即将到期，可用时间无变化，用蓝灯已经好几年了，忠实老用户，望尽快解决，谢谢！  账户邮箱为  geniuszhangke@163.com
![微信图片_20240617154819](https://github.com/getlantern/lantern-forum-cn/assets/173038303/391a324e-b7f1-4f56-930a-0e56f7252c0e)

**Comment by geniuszhangke** [comment]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@geniuszhangke 您好. 好的, 我帮您提交问题. 请给我们几天工作日时间. 尽快回复您. 请您之后留意这个帖子或邮箱. 谢谢您的耐心等待! 也非常感谢您支持蓝灯!
